geographic_msgs/CMakeLists.txt

```diff
@@ -40,11 +39,35 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   DEPENDENCIES unique_identifier_msgs geometry_msgs std_msgs builtin_interfaces
 )
 
+add_library(validation INTERFACE)
+add_library(${PROJECT_NAME}::validation ALIAS validation)
+include(GNUInstallDirs)
+add_subdirectory(include)
+
+include(CTest)
```

Comment:
CTest is included becauase it guarantees that BUILD_TESTING is defined.
https://cmake.org/cmake/help/latest/module/CTest.html

Reply:
I removed it because `ament_cmake` includes it instead. Transitive includes ftw.